filebeat/beater/filebeat.go

```diff
@@ -54,6 +54,14 @@ func New(b *beat.Beat, rawConfig *common.Config) (beat.Beater, error) {
 		return nil, err
 	}
 
+	if len(config.Prospectors) > 0 {
```

Comment:
Can you add some system tests with the old config options to make sure this works as expected? We have done this for previous renamings and I think it's pretty useful to make sure it actually works and be remembered that we break something, when we remove it.

Reply:
Of course, I was planning to do it in a dedicated PR, because right now all the tests use the old config options :)

Reply:
SGTM. Didn't realise it's all on the old config. Good to know.